scripts/collect_env.py

```diff
@@ -0,0 +1,281 @@
+import subprocess
+import platform
+import pkg_resources
+
+
+def command_exists(command):
+    """
+    Check if a command exists in the system
+
+    Args:
+        command (str): The command to check
+
+    Returns:
+        bool: True if command exists, False otherwise
+    """
+    try:
+        subprocess.run([command], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        return True
+    except FileNotFoundError:
+        return False
+
+
+def get_python_version():
+    """
+    Get the current Python version
+
+    Returns:
+        str: The current Python version
+    """
+    return platform.python_version()
+
+
+def get_os_version():
+    """
+    Get the current operating system version
+
+    Returns:
+        str: The current operating system version
+    """
+    return platform.platform()
+
+
+def is_python_package_installed(package_name):
```

Comment:
For this one you could use `dependency_exists` from `unstructured.utils`

https://github.com/Unstructured-IO/unstructured/blob/main/unstructured/utils.py#L50